pkg/util/containers/filter.go

```diff
@@ -99,46 +99,49 @@ func ResetSharedFilter() {
 }
 
 // NewFilter creates a new container filter from a two slices of
-// regexp patterns for a whitelist and blacklist. Each pattern should have
+// regexp patterns for a include list and exclude list. Each pattern should have
 // the following format: "field:pattern" where field can be: [image, name].
 // An error is returned if any of the expression don't compile.
-func NewFilter(whitelist, blacklist []string) (*Filter, error) {
-	iwl, nwl, nswl, err := parseFilters(whitelist)
+func NewFilter(includeList, excludeList []string) (*Filter, error) {
+	iwl, nwl, nswl, err := parseFilters(includeList)
 	if err != nil {
 		return nil, err
 	}
-	ibl, nbl, nsbl, err := parseFilters(blacklist)
+	ibl, nbl, nsbl, err := parseFilters(excludeList)
 	if err != nil {
 		return nil, err
 	}
 
 	return &Filter{
-		Enabled:            len(whitelist) > 0 || len(blacklist) > 0,
-		ImageWhitelist:     iwl,
-		NameWhitelist:      nwl,
-		NamespaceWhitelist: nswl,
-		ImageBlacklist:     ibl,
-		NameBlacklist:      nbl,
-		NamespaceBlacklist: nsbl,
+		Enabled:              len(includeList) > 0 || len(excludeList) > 0,
+		ImageIncludeList:     iwl,
+		NameIncludeList:      nwl,
+		NamespaceIncludeList: nswl,
+		ImageExcludeList:     ibl,
+		NameExcludeList:      nbl,
+		NamespaceExcludeList: nsbl,
 	}, nil
 }
 
-// NewFilterFromConfig creates a new container filter, sourcing patterns
-// from the pkg/config options
-func NewFilterFromConfig() (*Filter, error) {
-	whitelist := config.Datadog.GetStringSlice("container_include")
-	blacklist := config.Datadog.GetStringSlice("container_exclude")
-	if len(whitelist) == 0 {
+// newMetricFilterFromConfig creates a new container filter, sourcing patterns
+// from the pkg/config options, to be used only for metrics
```

Comment:
could you please document why we merge `container_exclude/include` and `container_exclude/include_metrics` here for future reference?

Reply:
Fixed